tests/contrib/aiohttp/test_request.py

```diff
@@ -31,6 +31,33 @@ async def test_full_request(patched_app_tracer, aiohttp_client, loop):
     assert "GET /" == request_span.resource
 
 
+async def test_full_request_w_mem_leak_prevention_flag(patched_app_tracer, aiohttp_client, loop):
+    config.aiohttp.disable_stream_timing_for_mem_leak = True
+    try:
+        app, tracer = patched_app_tracer
+        client = await aiohttp_client(app)
+        # it should create a root span when there is a handler hit
+        # with the proper tags
+        request = await client.request("GET", "/")
+        assert 200 == request.status
+        await request.text()
+        # the trace is created
+        traces = tracer.pop_traces()
+        assert 1 == len(traces)
+        assert 1 == len(traces[0])
+        request_span = traces[0][0]
+        assert_is_measured(request_span)
+
+        # request
+        assert "aiohttp-web" == request_span.service
+        assert "aiohttp.request" == request_span.name
+        assert "GET /" == request_span.resource
+    except Exception:
+        raise
+    finally:
+        config.aiohttp.disable_stream_timing_for_mem_leak = False
```

Comment:
What's the purpose of setting this to false at the end of the test if the default is false? (I'm wondering if there are some more assertions that are needed)